tests/tests/cpld_control_gefs

```diff
@@ -0,0 +1,243 @@
+
+# ufs.configure
+export coupling_interval_slow_sec=${DT_THERM_MOM6}
+export coupling_interval_fast_sec=${DT_ATMOS}
+
+# resolution dependent files
+#atm
+export FNALBC="'C384.snowfree_albedo.tileX.nc'"
+export FNALBC2="'C384.facsf.tileX.nc'"
+export FNTG3C="'C384.substrate_temperature.tileX.nc'"
+export FNVEGC="'C384.vegetation_greenness.tileX.nc'"
+export FNVETC="'C384.vegetation_type.tileX.nc'"
+export FNSOTC="'C384.soil_type.tileX.nc'"
+export FNSOCC="'C384.soil_color.tileX.nc'"
+export FNVMNC="'C384.vegetation_greenness.tileX.nc'"
+export FNVMXC="'C384.vegetation_greenness.tileX.nc'"
+export FNSLPC="'C384.slope_type.tileX.nc'"
+export FNABSC="'C384.maximum_snow_albedo.tileX.nc'"
+#marine
+export MOM6_INPUT=MOM_input_${OCNRES}.IN
+export MESH_ICE=mesh.mx${OCNRES}.nc
+export MESH_OCN=mesh.mx${OCNRES}.nc
+export CICE_GRID=grid_cice_NEMS_mx${OCNRES}.nc
+export CICE_MASK=kmtu_cice_NEMS_mx${OCNRES}.nc
+export MOM6_CHLCLIM=seawifs-clim-1997-2010.${NX_GLB}x${NY_GLB}.v20180328.nc
+export MOM6_FRUNOFF=runoff.daitren.clim.${NX_GLB}x${NY_GLB}.v20180328.nc
+export MOM6_RIVER_RUNOFF=True
+export MOM6_RESTART_SETTING=r
+export MOM6_DIAG_COORD_DEF_Z_FILE=interpolate_zgrid_26L.nc
+#
+
+#IAUs
+export READ_INCREMENT=.true.
+export RES_LATLON_DYNAMICS="'fv3_increment.nc'"
+export IAU_INC_FILES="'fv3_increment.nc'"
+export IAU_OFFSET=0
+export IAUFHRS='6'
+export IAU_DELTHRS=6
+#MOM6 IAU
+export ODA_INCUPD=True
+export ODA_INCUPD_NHOURS=0
+export ODA_TEMPINC_VAR="'t_pert'"
+export ODA_SALTINC_VAR="'s_pert'"
+export ODA_THK_VAR="'h_anl'"
+export ODA_INCUPD_UV=True
+export ODA_UINC_VAR="'u_pert'"
+export ODA_VINC_VAR="'v_pert'"
+
+#stochastic physics
+export STOCHINI=.true.
+export DO_SPPT=.true.
+export DO_SHUM=.false.
+export DO_SKEB=.true.
+export SKEB="0.8,-999,-999,-999,-999"
+export SPPT="0.56,0.28,0.14,0.056,0.028"
+export PERT_CLDS=.true.
+export PERT_MP=.false.
+export PERT_RADTEND=.false.
+export DO_CA=.true.
+export CA_GLOBAL=.false.
+export CA_SGS=.true.
+export NCA=1
+export NCELLS=5
+export NLIVES=12
+export NSEED=1
+export NCRACSEED=0.5
+export NTRHESH=18
+export CA_TRIGGER=.true.
+export NSPINUP=1
+export ISEED_CA=356398201
+# MOM6 stochastics
+export DO_OCN_SPPT=True
+export PERT_EPBL=True
+export OCN_SPPT="0.8,0.4,0.2,0.08,0.04"
+export EPBL="0.8,0.4,0.2,0.08,0.04"
+
+#Warm start
+export RUNTYPE=continue
+# ATM warm start
+export WARM_START=.true.
+export MAKE_NH=.false.
+export NA_INIT=0
+export EXTERNAL_IC=.false.
+export NGGPS_IC=.false.
+export MOUNTAIN=.true.
+#
+export NUDGE_QV=.false.
+export ATM_IGNORE_RST_CKSUM=.true.
+# ICE warm start
+export CICE_RUNTYPE=continue
+export CICE_USE_RESTART_TIME=.true.
+export MOM6_RESTART_SETTING=r
+
+#from ep5rt_ic/input.nml
+export NSTF_NAME=2,0,0,0,0
+export CDMBWD="5.0,5.0,1.0,1.0"
+export K_SPLIT=2
+export N_SPLIT=5
+export TAU=0.
+export RF_CUTOFF=10.
+export FV_SG_ADJ=450
+
+export PROGSIGMA=.true.
+export IOPT_TRS=2
+export IOPT_DIAG=1
+
+export IAER=2011
```

Comment:
@NickSzapiro-NOAA After discussion with GEFS team, let's keep this RT running with GOCART, but without aerosol radiative feedback. i.e., change

IAER=2011 to
IAER=1011

With this change, we can compare with the reforecasts but also test GOCART updates. 

Adding @bbakernoaa, @EricSinsky-NOAA, @BingFu-NOAA

Reply:
Done